pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -813,17 +828,40 @@ public void UpdateUI(bool selectionChanged = true)
                                 .GetSpectra(_peptides[index].Key, null, LibraryRedundancy.best).FirstOrDefault();
                         }
 
-                        var spectrumInfoR = LibraryRankedSpectrumInfo.NewLibraryRankedSpectrumInfo(spectrumInfo.SpectrumPeaksInfo,
-                                                                          transitionGroupDocNode.TransitionGroup.LabelType,
-                                                                          transitionGroupDocNode,
-                                                                          settings,
-                                                                          transitionGroupDocNode.Peptide.Target,
-                                                                          mods,
-                                                                          adducts,
-                                                                          types,
-                                                                          rankCharges,
-                                                                          rankTypes,
-                                                                          (spectrumInfo?.SpectrumHeaderInfo as BiblioSpecSpectrumHeaderInfo)?.Score);
+
+                        // Generates the object that will go into the property sheet
+                        var newProperties = new SpectrumProperties()
+                        {
+                            RetentionTime = string.Format(@"{0:F2}", spectrumInfo.RetentionTime),

```

Comment:
In the code below, we use the following to format retention time:
rt.Value.ToString(Formats.RETENTION_TIME)
and
public const String RETENTION_TIME = "0.##";
It would be nice to have these formatting decisions encapsulated into a single function, in case we decide to make them adjustable properties of the UI.